### DISCUSSION TITLE
Reject if the same file size is already downloaded.

### DISCUSSION THREAD

**Discussion by shotfirer** [Flexget/Flexget#3355] [Q&A]
Not sure if this is possible, but maybe someone has an idea how to do that?
Usually, the same TV episode is being released by several different release groups, but essentially it's the same file (exact file size). Currently, my config basically keeps upgrading the quality until I watch the episode (mark 'watched' on Trakt). But I wonder, is there any way to reject any new duplicates, which have the same file size as the already downloaded file?

**Comment by gazpachoking** [comment]
Probably fairly difficult. Tough to know the size of a thing unless your input plugin has it. Perhaps you could do something with crossmatch using `content_size` field against the filesystem. `content_size` doesn't get filled out until after filter phase though (unless your input plugin already provides it) so it could be tough.
